Booting/linux-bootstrap-4.md

```diff
@@ -480,7 +480,7 @@ Let's look at how this is implemented. First, we clear the buffer for the page t
 
 We put the address of `pgtable` with an offset of `ebx` (remember that `ebx` points to the location in memory where the kernel will be decompressed later) into the `edi` register, clear the `eax` register and set the `ecx` register to `6144`.
 
-The `rep stosl` instruction will write the value of `eax` to `edi`, add `4` to `edi` and decrement `ecx` by `1`. This operation will be repeated while the value of the `ecx` register is greater than zero. That's why we put `6144` or `BOOT_INIT_PGT_SIZE/4` in `ecx`.
+The `rep stosl` instruction will write the value of `eax` to `edi`, add `4` to the memory location where `edi` points to and decrement `ecx` by `1`. This operation will be repeated while the value of the `ecx` register is greater than zero. That's why we put `6144` or `BOOT_INIT_PGT_SIZE/4` in `ecx`.
```

Comment:
`stosl` => `mov [edi], eax; add edi, 4`. You add the correct description to an incorrect `edi`. It should be the first `edi`. See details in <https://www.felixcloutier.com/x86/stos:stosb:stosw:stosd:stosq>

Reply:
Oh my god... pardon me.

Reply:
Fixed, force-pushed.